### ISSUE TITLE
Rewind no longer shows frames on 2.4

### ISSUE THREAD

**Issue by viniciusccarvalho** [OpenEmu/OpenEmu#4956]
After getting the latest version 2.4, I noted that rewind altough working, now shows a black frame, making it hard to figure out when to stop rewinding. 
I noted this on Genesis, SNES, Sega cd cores, all are no longer showing the frames
# Debugging Information

- OpenEmu Version: 2.4
- macOS Version: ???

**Comment by clobber** [comment]
Yes. GenesisPlus and Snes9x show a black screen during rewind or after stepping backwards. Broken in e62151b/OpenEmuKit-08af8ff

**Comment by SoloBigGangster** [comment]
Me too, I had the same problem as well when trying to play while doing rewind, which resulted in a black screen.

**Comment by SoloBigGangster** [comment]
Also, the version I use is version 2.4.1.

**Comment by Beancity** [comment]
Had this same issue too. Trying to play a SNES game presumably w/ SNES9X.

**Comment by SoloBigGangster** [comment]
I hope this gets fixed in the next update version.
